pyscf/df/outcore.py

```diff
@@ -189,7 +197,11 @@ def process(sh_range):
         return dat
 
     feri = _create_h5file(erifile, dataname)
-    for istep, dat in enumerate(lib.map_with_prefetch(process, shranges)):
+
+    # Cannot use lib.map_with_prefetch with trsm because
```

Comment:
Create two buffers and swap the two buffers, as the incore code does.

Reply:
OK!